src/lib/services-manager/clients/services_manager.rb

```diff
@@ -292,6 +317,15 @@ def handle_dialog
         ServicesManagerTarget.default_target = new_default_target
       end
 
+      def handle_table
+        if @prev_service == current_service
+          toggle_service
```

Comment:
so when you click on already selected service it is enabled/disabled? I think that it also ignore start on demand. I believe whole this toggle_service should be removed as it is no longet two state enablement.

Reply:
Or we can cycle throw all possible values.